app/views/brexit_landing_page/show_B.html.erb

```diff
@@ -0,0 +1,112 @@
+    <% end %>
+    <%= render "govuk_publishing_components/components/share_links", {
+      title: "Share this page",
+      columns: true,
+      links: [
+        {
+          href: "https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.gov.uk%2Fbrexit",
+          text: hidden_text + "Facebook",
+          icon: "facebook"
+        },
+        {
+          href: "https://twitter.com/share?url=https%3A%2F%2Fwww.gov.uk%2Fbrexit",
+          text: hidden_text + "Twitter",
+          icon: "twitter"
+        },
+        {
+          href: "mailto:?body=https://www.gov.uk/brexit/&subject=Get ready for Brexit - GOV.UK",
+          text: hidden_text + "Email",
+          icon: "email"
+        },
+        {
+          href: "https://api.whatsapp.com/send?text=https://www.gov.uk/brexit",
+          text: hidden_text + "WhatsApp",
+          icon: "whatsapp"
+        },
+        {
+          href: "http://www.linkedin.com/shareArticle?url=https%3A%2F%2Fwww.gov.uk%2Fbrexit&title=Get%20ready%20for%20Brexit%20-%20GOV.UK",
+          text: hidden_text + "LinkedIn",
+          icon: "linkedin"
+        },
+      ]
+    } %>
+  </div>
+</div>
+
+<div class="full-page-width-wrapper full-page-width-wrapper--landing-page">
+  <div class="govuk-grid-row">
+    <div class="govuk-body govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: t('brexit_landing_page.all_brexit_information'),
+        padding: true,
+      } %>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: t('brexit_landing_page.finder_heading'),
+        heading_level: 3,
+        font_size: 19,
+        padding: true
+      } %>
+      <%= render "components/topic-list", {
+        items: presented_taxon.supergroup_sections
+      } %>
+      </div>
+    </div>
+
+  <div class="landing-page__email-wrapper">
+    <svg  class="app-c-email-link__icon" xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 459.334 459.334">
+      <path fill="black" d="M177.216 404.514c-.001.12-.009.239-.009.359 0 30.078 24.383 54.461 54.461 54.461s54.461-24.383 54.461-54.461c0-.12-.008-.239-.009-.359H175.216zM403.549 336.438l-49.015-72.002v-89.83c0-60.581-43.144-111.079-100.381-122.459V24.485C254.152 10.963 243.19 0 229.667 0s-24.485 10.963-24.485 24.485v27.663c-57.237 11.381-100.381 61.879-100.381 122.459v89.83l-49.015 72.002a24.76 24.76 0 0 0 20.468 38.693H383.08a24.761 24.761 0 0 0 20.469-38.694z"/>
+    </svg>
+    <h3 class="govuk-heading-s landing-page__email-title"><%= t('brexit_landing_page.email_intro') %></h3>
+    <%= link_to(t('brexit_landing_page.email'), "/email-signup/?topic=#{presented_taxon.base_path}", {
+      class: "govuk-body govuk-link",
+      data: {
+        module: "track-click",
+        "track-category": "emailAlertLinkClicked",
+        "track-action": presented_taxon.base_path
+      }
+    }) %>
+  </div>
+</div>
```

Comment:
Is this view basically the same as variant A at the moment? If so I wonder if we want to just using the same view until divergence is sufficiently significant that it warrants complete new views. Otherwise in the meantime we have to maintain two pages of near identical HTML.

Reply:
It's identical at present, I thought for the testing and logic side of things it would be better to have two fully functioning views but I suppose I can remove this one and point both renders to `show_a` for now until `show_b` is implemented.